rainforest-cli.go

```diff
@@ -236,10 +237,10 @@ func main() {
 		},
 		{
 			Name:        "rm",
-			Usage:       "Remove an RFML test locally and remotely",
+			Usage:       "Remove an RFML test locally and remotely (if token is specified)",
```

Comment:
I think we should force them to have a token. Otherwise people are encouraged to delete their tests locally but not remotely. If they just want to remove a local file, they don't need us to do that for them :D